docs/maintainers/crate_release.md

```diff
@@ -37,17 +62,20 @@ and the code owners of each repository.
 ...
 ```
 
-3. Update the version field in Cargo.toml in the root of the crate's
+3. Update the version field in `Cargo.toml` in the root of the crate's
    repository.
```

Comment:
Shouldn't this be "folder" instead of "repository", to be in line with the change on line 41?

Reply:
Great observation, thank you! I fixed it and pushed the changes.